### ISSUE TITLE
Integration in Home Assistant via iframe disables HA return button

### ISSUE THREAD

**Issue by Bhoft** [CDrummond/lms-material#862]
Hi I just found your beautiful extension for the LMS and I wanted it to integrate into my home assistant via iframe.

The issue is whenever the iframe with the lms-material plugin is embeeded in a Sub-view of an Home assitant dashboard the return button in the main navigation of HA isn't working any more.

![grafik](https://github.com/CDrummond/lms-material/assets/1777147/8f192240-8b27-418f-9693-9035599b8ee1)

Its not a normal link but a button so i guess `history.back()` or something is triggered when the button is clicked.

When included on a normal dashboard view all main navigations links to other sites of the dashboard are still working.
Also navigation via buttons on the dashboard itself also works without any issue.
Only on subviews the back-button isn't working.

Do you have perhaps an idea what this could be or what i could perhaps check?

This is how i added the page to Homeassistant:
```
type: iframe
url: >-
  http://192.168.5.86:9000/material/?layout=mobile&page=now-playing&player=00:04:20:2e:27:c3
aspect_ratio: 100%
view_layout:
  place-self: stretch stretch
  justify-self: stretch
```

What I tested:
- The dashboard back button works for "normal" websites included via iframe.
- the back button works when the lms-material player isn't fully loaded yet.
- the back button triggers when clicking multiple times really fast.
- the issue happens on mobile and desktop browsers
- on mobile phone the back button gesture also wouldn't change back to the previous view/url
- the browser back button on desktop browser also doesn't react on click
- also happens with the desktop layout of lms-material

I see no javascript error in the browser console. 

**Do you maybe prevent/change the browser history?**

Perhaps you or someone else is also using Home assitant and might have an idea what causes this?

Example of an Home assistent test dashboard with Home and a Subview called sub-page. 
On both Home and the sub-page the player is included via iframe.

```
views:
  - title: Home
    cards:
      - type: iframe
        url: >-
          http://192.168.5.86:9000/material/?layout=mobile&page=now-playing&player=00:04:20:2e:34:18
        aspect_ratio: 100%
        view_layout:
          place-self: stretch stretch
          justify-self: stretch
      - show_name: true
        show_icon: true
        type: button
        tap_action:
          action: navigate
          navigation_path: /test-dashboard/sub-page
        icon: mdi:button-pointer
        hold_action:
          action: navigate
          navigation_path: /test-dashboard/sub-page
  - title: sub-page
    path: sub-page
    subview: true
    icon: mdi:play
    cards:
      - type: iframe
        url: >-
          http://192.168.5.86:9000/material/?layout=mobile&page=now-playing&player=00:04:20:2e:34:18
        aspect_ratio: 100%
        view_layout:
          place-self: stretch stretch
          justify-self: stretch
      - show_name: true
        show_icon: true
        type: button
        tap_action:
          action: navigate
          navigation_path: /test-dashboard/0
        icon: mdi:button-pointer
        hold_action:
          action: navigate
          navigation_path: /test-dashboard/0
```

**Comment by CDrummond** [comment]
Material intercepts the browser's back button to navigate back in its views, etc. I _think_ this is what might be causing the issue here.

I've just updated the code to add support for a URL query parameter to stop this happening. So, if you used `http://server:9000/material/` using `http://server:9000/material/?dontTrapBack` should then stop this. You need to be using a checkout of git/master to try this.

**Comment by Bhoft** [comment]
Thank you very much for the quick response.

Can you help me how i can use the git/master version in my LMS?
Because I only add the repository by addin this https://raw.githubusercontent.com/CDrummond/lms-material/master/public.xml which installs the 4.4.0 release.
And every js file is minified and also combined (e.g. I can't find a utils.js file iin the installed plugin) 

OK i guess i found out. If I manually checkout the repository locally and exchanged the "MaterialSkin" folder in my LMS Plugins directory with the fresh one it still works. So no manual installation process is needed.

And i can say the fix worked perfectly!
Thank you very much.

Could you create a release version from that so that the automatic update from LMS could still be used.

**Comment by CDrummond** [comment]
Thanks for the confirmation. Fix will be in 4.4.1 - I don't make a release per fix, sorry. But will probably release at the weekend.
